kanren/tests/test_cons.py

```diff
@@ -0,0 +1,110 @@
+from functools import reduce
+from itertools import chain, cycle
+from collections import Iterable, OrderedDict
+
+from kanren.cons import cons, ConsPair, car, cdr, is_cons, is_null
+
+
+def assert_equal(a, b):
```

Comment:
What's the purpose of this? I think it's more readable (and shorter) to use the `assert expr1 == expr2` pytest syntax rather than defining a method.

Reply:
That's left-over from something else, as well.  One sec.